app/views/unattended/provisioning_templates/provision/preseed_default.erb

```diff
@@ -97,7 +97,16 @@ d-i time/zone string <%= host_param('time-zone') || 'UTC' %>
 
 # NTP
 d-i clock-setup/ntp boolean true
-d-i clock-setup/ntp-server string <%= host_param('ntp-server') || '0.debian.pool.ntp.org' %>
+<% if host_param('ntp-server') -%>
+<% if host_param('ntp-server').is_a?(String) -%>
+d-i clock-setup/ntp-server string <%= host_param('ntp-server') %>
+<% else -%>
+# host_param('ntp-server') is an array, but preseed only permits one value
+d-i clock-setup/ntp-server string <%= host_param('ntp-server').first %>
```

Comment:
This surprised me, but you're right that it only takes a single value: https://salsa.debian.org/installer-team/clock-setup/-/blob/master/debian/clock-setup.postinst?ref_type=heads#L25

Now reading this I also wonder if providing a default is good here: we ignore the DHCP provided option. However, that's not in scope for this PR so :+1: for keeping it hardcoded.